app/grpc/HelloWorldRouter.java

```diff
@@ -1,4 +1,4 @@
-package controllers;
+package grpc;
```

Comment:
I understood from the Play team that nowadays it might be more common to use 'traditional' packages (`com.lightbend.grpc.example.routers`?) rather than `controllers` etc?

Reply:
I though that "avoid controllers for the grpc thing since it is not a controller but router" was the idea. Should this be routers? I'm ok with that

Reply:
i got the impression that the packages like 'controllers' were out of vogue entirely - @marcospereira @TimMoore ?